src/server/worldserver/worldserver.conf.dist

```diff
@@ -2064,6 +2064,14 @@ GM.InWhoList.Level = 3
 
 GM.StartLevel = 1
 
+#
+#    GM.AllowAchievements
+#        Description: Allow GM accounts to have achievements.
```

Comment:
maybe explain a bit more in detail that setting this to 1 but having ".gm on" will not give gms achievements

Reply:
Maybe something like the wording in the comment in `AchievementMgr.cpp` ?